tox.ini

```
@@ -46,12 +46,16 @@ extras =
 commands_pre =
     find {toxinidir} -type f -not -path '{toxinidir}/.tox/*' -path '*/__pycache__/*' -name '*.py[c|o]' -delete
 commands =
-    unit: pytest test/unit/ --cov={toxinidir}/molecule/ --no-cov-on-fail {posargs}
-    functional: pytest test/functional/ {posargs}
-    lint: flake8
-    lint: yamllint -s test/ molecule/
+    unit: python -m pytest test/unit/ --cov={toxinidir}/molecule/ --no-cov-on-fail {posargs}
+    functional: python -m pytest test/functional/ {posargs}
+    lint: python -m flake8
+    lint: python -m yamllint -s test/ molecule/
+# avoid possible undesired warnings when used with site-packages:
 whitelist_externals =
     find
+    # https://github.com/alanhamlett/pip-update-requirements/issues/24
+    pur
```

This one is not relevant anymore
```suggestion
    pur
```